src/style/form/form.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import './style-form.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { colors } from '../colors';
+import { components } from '../typography/typography';
+import { Label } from '../typography/LargeLabel';
+interface FormProps {
+  error?: boolean;
+  caption?: string;
+  label?: string;
+  iconError?: IconDefinition;
+  icon?: IconDefinition;
+  password?: boolean;
+}
+
+export const styleError = {
+  fontFamily: components.family.primary,
+  color: colors.neutral.neutralDark,
+}
+
+export const Form = ({error, label, icon, password, caption, iconError}: FormProps) => {
+  const [focused, setFocused] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.target.value);
+  };
+
+  let border;
+
+  if (focused) {
+    border = `2px solid ${colors.neutral.neutralLight}`;
+  } else if (error) {
+    border = `1px solid ${colors.feedback.feedbackError}`;
+  } else if (inputValue) {
+    border = `1px solid ${colors.neutral.neutralLight}`;
+  } else {
+    border= `1px solid`;
```

Comment:
vc pode usar o `formatOnSave` do vscode para aplicar os padroes do seu prettier automaticamente para vc. Da uma olhada [nesse tutorial](https://geekflare.com/auto-format-in-vs-code/#:~:text=3.%20Type%20Formatter%20in%20the%20search%20bar%20and%20select%20Prettier%20on%20the%20Editor%3ADefault%20Formatter%20tab.%C2%A0)